### PR TITLE
Contact Form

### LINKED ISSUE #89: Entering new page from an event removes the calendar
The calendar disappears when going back from the contact form to the tab. 
Here's a GIF for clarification:
ed74a42c579e33b09d0f3393f92d2934

### PR DESCRIPTION
**Pull request by tonitajger**
Adding **contact form** to the app which sends an email to the sender and the chosen contact. Linking through events to the specific contact is possible. Also fixes the bug where the calendar, news and notifications doesn't load when returning from pages deeper than 1 level (closes #89 ).

### CODE REVIEW

File: www/contact.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!--TODO: erase the text form when message has been sent, check calender bug, add information about what you're sending -->
+<div class="pages">
```
Comment (FredrikLastow) [inline www/contact.html]: This pages div is unnecessary and maybe also the comment?

File: config.xml [hunk 2]
```diff
@@ -60,6 +60,8 @@
     <preference name="AutoHideSplashScreen" value="false" />
     <preference name="phonegap-version" value="cli-7.1.0" />
     <preference name="HeaderColor" value="#eb7125" />
+    <engine name="android" spec="^6.2.3" />
+    <plugin name="cordova-plugin-dialogs" spec="~1.3.3" />
```
Comment (FredrikLastow) [inline config.xml]: Think these needs to be downgraded to work with PhoneGap build. Same goes for package-lock.json and package.json, I think. @navro

File: www/js/index.js [hunk 3]
```diff
@@ -21,9 +22,9 @@ Template7.global = {
 var $$ = Dom7;
 
 // API URLS
-const BASE_URL = 'https://stage.fsektionen.se'
-const API = 'https://stage.fsektionen.se/api'
-const AC_URL = 'wss://stage.fsektionen.se/cable'
+const BASE_URL = 'https://fsektionen.se'
+const API = BASE_URL + '/api'
+const AC_URL = 'wss://fsektionen.se/cable'
```
Comment (FredrikLastow) [inline www/js/index.js]: Change back to the stage api url

File: www/index.html [hunk 4]
```diff
@@ -605,8 +607,8 @@
             <div class="card-footer">
               <div class="event-contact">
                 Vid tekniska problem med anmälan, hör vänligen av dig till:
-                <a href="https://www.fsektionen.se/kontakter/1" class="external" target="_system">
-                  <i class="fa fa-envelope" aria-hidden="true"></i> Spindelmännen
+                <a href="contact.html?id=1">
+                  <i class="fa fa-envelope" aria-hiddesn="true"></i> Spindelmännen
```
Comment (FredrikLastow) [inline www/index.html]: A weird "s" sneaked its way in there. Should be "aria-hidden" of course.

File: config.xml (hunk 2 above)
Comment (navro) [reply config.xml]: Yes, please use the versions specified on master :)

File: package.json [hunk 5]
```diff
@@ -7,20 +7,20 @@
...
-      "cordova-plugin-headercolor": {},
-      "cordova-plugin-dialogs": {}
+      "cordova-plugin-headercolor": {}
     }
   },
   "dependencies": {
-    "cordova-android": "^6.4.0",
-    "cordova-plugin-dialogs": "^2.0.1",
+    "cordova-android": "^6.2.3",
+    "cordova-plugin-dialogs": "~1.3.3",
```
Comment (navro) [inline package.json]: Same here

**Comment by navro** [comment]
Looks great excepts for some small comments :+1:

**Comment by FredrikLastow** [comment]
Well done! <3

**Pull request merged**